src/codemap_details/Components/Query.js

```diff
@@ -1,6 +1,56 @@
 import React from "react";
 import styles from "../CSS/Query.module.css";
+import query from "../assets/query.png";
+import { Jumbotron, Col, Row, Image, Container } from "react-bootstrap";
 
 export default function Query() {
-  return <React.Fragment />;
+  return (
+    <div>
+      <Jumbotron className={styles.jumbo} fluid>
+        <Container className={styles.container}>
+          <h1 className={styles.head}>
+            What if you have a <span className={styles.query}>Query</span>?
+          </h1>
+          <Row>
+            <Col lg={6}>
+              <p className={styles.para}>
+                Duis aute irure dolor in reprehenderit in voluptate velit esse
+                cillum dolore eu fugiat nulla pariatur.
+                <br />
+                <br />
+                Excepteur sint occaecat cupidatat non proident, sunt in culpa
+                qui officia deserunt mollit anim id est laborum.
+              </p>
+              <hr className={styles.line} />
+            </Col>
+            <Col lg={6}>
+              <Image src={query} className={styles.img} fluid />
+            </Col>
+          </Row>
+        </Container>
+        <Container className={styles.small} fluid>
+          <h1 className={styles.head} style={{ textAlign: "center" }}>
```

Comment:
@prachibansal2000 Please avoid inline styling. It creates troubles while we scale up.

Reply:
ok